src/chart/draw/consolidateData.js

```diff
@@ -10,7 +10,7 @@ export default function consolidateData(raw) {
     if (this.filters.length) {
         this.filters.forEach(filter => {
             this.filtered_data = this.filtered_data.filter(d => {
-                return filter.val === 'All'
+                return filter.all === true && filter.index === 0
```

Comment:
@pburnsdata Does `filter.index` change when the user updates a subsetter input? What code controls that update?

Reply:
yes, just like val changes - the code that controls it is below